components/org.wso2.identity.webhook.common.event.handler/src/main/java/org/wso2/identity/webhook/common/event/handler/internal/component/EventHookHandlerServiceComponent.java

```diff
@@ -92,6 +97,11 @@ protected void activate(ComponentContext context) {
                 bundleContext.registerService(AbstractEventHandler.class.getName(),
                         new RegistrationEventHookHandler(), null);
             }
+            if (isTokenEventHandlerEnabled != null && isTokenEventHandlerEnabled
+                    .equalsIgnoreCase(Boolean.TRUE.toString())) {
```

Comment:
Can't we read this directly as a boolean ? if it's something else it will return false right